pipeline/scraper/logger.js

```diff
@@ -45,4 +45,4 @@ module.exports = {
     debug: function() {
         log(DEBUG, arguments);
     }
-};
+};
```

Comment:
needs EOF new line.

Probs VS Code automatically removing it, run through ESLint using ```--fix```